circle.yml

```diff
@@ -9,5 +9,6 @@ dependencies:
 test:
   override:
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
-    - ./conda_rpms/tests/integration/gitenv_create_rpmbuild.sh
-    - ./conda_rpms/tests/integration/gitenv_build_rpms.sh
+    #    - ./conda_rpms/tests/integration/gitenv_create_rpmbuild.sh
+    #- ./conda_rpms/tests/integration/gitenv_build_rpms.sh
```

Comment:
Yup, makes sense these integration tests are failing anyways :+1:

Reply:
Argh sorry. I only actually did this cause I didn't want to have to wait for these tests to run. I'll un-comment these so it's back to how it was